changelog/fix-finish-setup-link

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: dev
```

Comment:
This doesn't look like a dev change but a user-facing one.

Reply:
It is a dev note because the JCB has not been released yet.

Reply:
I could be wrong, but AFAIK "dev" changes refer to ones related to developer experience like tooling etc.